gdal-sys/src/lib.rs

```diff
@@ -5,4 +5,3 @@
 #![allow(rustdoc::bare_urls)]
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
-include!(concat!(env!("OUT_DIR"), "/docs_rs_helper.rs"));
```

Comment:
This must have been satisfying to eliminate! 😄

Reply:
yes :D